packages/manager/modules/pci/src/projects/new/payment/components/credit/controller.js

```diff
@@ -0,0 +1,72 @@
+import head from 'lodash/head';
+
+import { PREDIFINED_AMOUNT_SEQUENCE } from './constants';
+
+export default class PciProjectNewPaymentCreditCtrl {
+  constructor() {
+    // other attributes
+    this.predifinedAmounts = [];
+    this.otherAmount = {
+      value: null,
+      radio: false,
+      getterSetter: (...args) => {
```

Comment:
I think we should distinguish `get` and `set` and not have both in one function.

Reply:
I don't think angularJS getterSetter model options will work with 2 distinguished methods